test/go-gossipsub.js

```diff
@@ -2,7 +2,9 @@
  * These tests were translated from:
  *   https://github.com/libp2p/go-libp2p-pubsub/blob/master/gossipsub_test.go
  */
-const { expect } = require('chai')
+const chai = require('chai')
+chai.use(require('dirty-chai'))
```

Comment:
nit: use aegir exported chai instead

Reply:
That would be my preference too, but it's not used anywhere else in this module so I did the manual `.use` to be consistent with the other tests.